xml/System.Windows.Forms/SystemInformation.xml

```diff
@@ -3405,19 +3399,19 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets a value indicating whether the operating system natively supports a mouse wheel.</summary>
+        <summary>Gets a value indicating whether a mouse with a mouse wheel is installed.</summary>
         <value>
-          <see langword="true" /> if the operating system natively supports a mouse wheel; otherwise, <see langword="false" />.</value>
+          <see langword="true" /> if a mouse with a mouse wheel is installed; otherwise, <see langword="false" />.</value>
         <remarks>
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- Mouse wheel operations that occur through a <xref:System.Windows.Forms.Control> object work even if the operating system does not natively support the wheel.  
-  
+ The <xref:System.Windows.Forms.SystemInformation.NativeMouseWheelPresent%2A> property indicates whether the system recognizes a mouse wheel on an installed mouse.  
```

Comment:
This xref doesn't resolve. Is it NativeMouseWheelSupport?

Reply:
They are, sorry. :)